atc/resource/run_script.go

```diff
@@ -52,9 +52,11 @@ func (resource *resource) runScript(
 	}
 
 	if recoverable {
-		result, err := resource.container.Property(resourceResultPropertyName)
-		if err == nil {
-			return json.Unmarshal([]byte(result), &output)
+		result, _ := resource.container.Properties()
+		for prop, code := range result {
```

Comment:
Hey @aledeganopix4d 
Does it make sense to use a key lookup rather than iterating over the map ?